.gitignore

```diff
@@ -10,6 +10,7 @@ drupal
 .idea
 build
 *.bak
+*.phar
```

Comment:
What file are you needing to ignore here? There shouldn't be any phars in the root directory.

Reply:
I found these get generated during local testing of phar builds.

Reply:
An extra .gitignore line won't hurt anything, but it might indicate something wrong with your local, since phars should never exist outside of the `build` dir following https://github.com/acquia/cli/pull/458